app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    splitted_command = command.split()
+    original_file = splitted_command[1]
```

Comment:
You should check the length of split command before retrieve values
It should be exactly 3